gradle/dependencies.gradle

```diff
@@ -1,5 +1,5 @@
 def versions = [
-    apolloVersion    : '0.2.3-SNAPSHOT',
+    apolloVersion    : '0.3.0',
```

Comment:
This variable is used for the sample and integration modules so it should stay as `0.2.3-SNAPSHOT` because v0.3.0 doesn't exist.